### PR TITLE
Use beeswax for Hive monad

### PR DESCRIPTION
**Pull request by shmookey**

### CODE REVIEW

**Comment by vineethvarghese** [comment]
version updated is needed otherwise :+1:

**Comment by stephanh** [comment]
So now we have got 2 hive monads? One in beeswax and one in ebenezer? And nothing even ensures that we only use the beeswax ones now?

**Comment by shmookey** [comment]
@vineethvarghese Good point, done. 
@stephanh Correct (see gitter channel)

File: maestro-scalding/src/test/scala/au/com/cba/omnia/maestro/task/ViewExecutionSpec.scala [hunk 1]
```diff
@@ -29,7 +29,7 @@ import au.com.cba.omnia.thermometer.fact.PathFactoids._
 import au.com.cba.omnia.thermometer.hive.ThermometerHiveSpec
 
 import au.com.cba.omnia.ebenezer.ParquetLogging
-import au.com.cba.omnia.ebenezer.scrooge.hive._
+import au.com.cba.omnia.beeswax._
```
Comment (stephanh) [inline maestro-scalding/src/test/scala/au/com/cba/omnia/maestro/task/ViewExecutionSpec.scala]: Can we make this a qualified import? Is there lots of stuff under Beeswax that we are importing?

File: version.sbt [hunk 2]
```diff
@@ -12,7 +12,7 @@
 //   See the License for the specific language governing permissions and
 //   limitations under the License.
 
-version in ThisBuild := "2.17.0"
+version in ThisBuild := "2.17.1"
```
Comment (stephanh) [inline version.sbt]: Lets make it a minor version bump.

**Comment by stephanh** [comment]
:+1:

**Pull request merged**